pmm-app/src/pmm-dbaas/components/DBCluster/AddDBClusterModal/AddDBClusterModal.styles.ts

```diff
@@ -1,11 +1,32 @@
 import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
 
-export const styles = {
+export const getStyles = ({ spacing, typography, colors }: GrafanaTheme) => ({
   stepProgressWrapper: css`
     overflow: hidden;
 
     div[class$='-current'] {
       overflow: auto;
     }
   `,
-};
+  warningIcon: css`
+    fill: red;
```

Comment:
Can use danger color from grafana and theme spacing bellow

Reply:
good point, fixed